hudi-common/src/main/java/org/apache/hudi/common/fs/HoodieWrapperFileSystem.java

```diff
@@ -79,22 +84,51 @@ public static void setMetricsRegistry(Registry registry, Registry registryMeta)
     METRICS_REGISTRY_META = registryMeta;
   }
 
+  // Configs which controls IO buffering and buffer sizes
+  private static final String CONFIG_IO_BUFFER_PREFIX = "hoodie.fs.io.buffer";
```

Comment:
@prashantwason Can we move this to a config class like "HoodieWrapperFileSystemConfig" like this -> https://github.com/apache/hudi/tree/master/hudi-common/src/main/java/org/apache/hudi/common/config ?
Then you can use the same config in the hudi-client-common and here and avoid issues when the name scope has changed in one place and hasn't changed in another. 
Let's keep the same names you have the HoodieWriteConfig.

Reply:
+1. would be nice to avoid duplicated configs.

Reply:
Done.